assets/js/components/adminbar/AdminBarWidgets.js

```diff
@@ -131,7 +134,9 @@ export default function AdminBarWidgets() {
 					</Fragment>
 				) }
 
-				{ ( ! analyticsModuleConnected || ! analyticsModuleActive ) && (
+				{ ( ( analyticsModuleAvailable &&
```

Comment:
Whoops - this comment has disappeared from the "Files changed" view and I've only just spotted it, hence addressing it last.

My answer to this is along the same lines as the comment below - this check for `analyticsModuleAvailable` is required in conjunction with `! analyticsModuleConnected || ! analyticsModuleActive`.

This is because we only want to handle the "not connected/active" case when the module _does_ exist, only showing the Analytics-related components when Analytics does exist to be connected/activated.